radicle-httpd/src/api/v1/projects.rs

```diff
@@ -401,6 +450,38 @@ async fn issue_handler(
     Ok::<_, Error>(Json(issue))
 }
 
+/// Returns the name part of a path string.
+fn name_in_path(path: &str) -> &str {
+    match path.rfind('/') {
+        Some(idx) => {
+            if idx == path.len() - 1 {
+                ""
+            } else {
+                &path[idx + 1..]
+            }
+        }
+        None => path,
+    }
```

Comment:
I think this can be written as:
```suggestion
    match path.rsplit('/').next() {
        Some(name) => name,
        None => path,
    }
```